src/main/java/com/Elessar/app/Main.java

```diff
@@ -1,49 +1,101 @@
 package com.Elessar.app;
 
-import com.Elessar.app.server.MyServer;
+import com.Elessar.app.client.MyClient;
+import com.Elessar.app.client.MyClientServer;
 import com.Elessar.app.util.MetricManager;
-import com.Elessar.database.MongoDB;
-import com.Elessar.database.MyDatabase;
-import com.example.tutorial.Addressbook.Person;
-import com.mongodb.client.MongoClients;
-import java.io.*;
-import java.util.logging.Logger;
-import java.util.logging.Level;
-
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
+import java.util.Random;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
 
 /**
- * Created by Hans on 1/7/19.
+ * Created by Hans on 2/27/19.
  */
-
 public class Main {
-    public static void main(String[] args){
-        Logger mongoLogger = Logger.getLogger( "org.mongodb.driver" );
-        mongoLogger.setLevel(Level.SEVERE); // e.g. or Log.WARNING, etc.
-        final MetricManager metricManager = new MetricManager("ServerMetric", 0);
-        final MyDatabase db = new MongoDB(MongoClients.create("mongodb://localhost:27017").getDatabase("myDB"), metricManager);
-        final MyServer server = new MyServer("localhost", 9000, db, metricManager);
-        server.run();
-        //testProtoBuf();
-    }
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
+    /**
+     *
+     * @param args [0] username for user1
+     *             [1] port number for user1
+     *             [2] username for user2
+     *             [3] port number for user2
+     *             [4] total number of messages to send
+     */
+    public static void main(String[] args) throws Exception {
+        final String serverURL = new StringBuilder().append("http://127.0.0.1:9000").toString();
+        final MetricManager metricManager = new MetricManager("ClientMetric", 0);
```

Comment:
You need to choose a realistic maxEntryCount, especially when we are doing perf test here. Writting to disk too frequently will negatively impact overall performance